meetup/tests.py

```diff
@@ -219,5 +220,5 @@ def test_parsed_response_contains_event_datetime(self):
     self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('datetime'), 1458730800000)
 
   def test_parsed_response_contains_lat_and_long(self):
-    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('lat'), 40.7599983215332)
-    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('lon'), -73.98999786376953)
+    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('lat'), 40.75501251220703)
```

Comment:
I don't understand why this changed. Is it possible your tests are talking to a real API?